app/components/satis/menu_item/component.html.slim

```diff
@@ -1,9 +1,13 @@
 li.rounded-sm.px-3.py-1.hover:bg-gray-100.dark:hover:bg-gray-200.flex.items-center data-controller="satis-menu" data-action="mouseover->satis-menu#show mouseleave->satis-menu#hide"
   a.cursor-pointer.py-1.w-full.text-left.flex.items-center.outline-none.focus:outline-none href=item.link *item.link_attributes
+    - if item.type == :form
```

Comment:
Should this than still be named form, or is `:custom` better?
I'd rename file_name to path.

Reply:
Yes sure, Will change it, Thank you!